src/ontology/mondo-edit.obo

```diff
@@ -412091,6 +412151,22 @@ synonym: "immunodeficiency 75" EXACT [OMIM:619126]
 xref: OMIM:619126 {source="MONDO:equivalentTo"}
 is_a: MONDO:0021094 {source="OMIM:619126"} ! immunodeficiency disease
 
+[Term]
+id: MONDO:0030859
+name: COACH syndrome 2
+synonym: "COACH2" EXACT ABBREVIATION [OMIM:619111]
+xref: OMIM:619111 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0030860
+name: neuronopathy, distal hereditary motor, type 5c
+synonym: "Dhmn5C" EXACT [OMIM:619112]
+synonym: "HMN5C" EXACT ABBREVIATION [OMIM:619112]
+synonym: "Spinal Muscular Atrophy, Distal, Type 5C" EXACT [OMIM:619112]
+xref: OMIM:619112 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```

Comment:
classify under dHMN 

consider adding a grouping dHMN5, we have 5A and 5B. btw, 5B may be conflated, can you look into it

Reply:
I created a grouping class: MONDO_0100350 'neuronopathy, distal hereditary motor, type 5' that is equiv to [Orphanet:139536 Distal hereditary motor neuropathy type 5 ](https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http%3A%2F%2Fwww.orpha.net%2FORDO%2FOrphanet_139536)

Orphanet:139536 was MONDO:equvialentTo MONDO_0013884 'neuronopathy, distal hereditary motor, type 5B' and I changed it to MONDO:subClassOf